tests/integration_tests/test_wandb.py

```diff
@@ -0,0 +1,77 @@
+from unittest import mock
+
+import optuna
+from optuna.integration import WeightsAndBiasesCallback
+
+
+def _objective_func(trial: optuna.trial.Trial) -> float:
+
+    x = trial.suggest_uniform("x", low=-10, high=10)
+    y = trial.suggest_loguniform("y", low=1, high=10)
+    return (x - 2) ** 2 + (y - 25) ** 2
+
+
+@mock.patch("optuna.integration.wandb.wandb")
+def test_run_initialized(wandb: mock.MagicMock) -> None:
+
+    wandb.run = None
```

Comment:
Please correct me if I asking a strange question, but is this line necessary?

Reply:
Huh, following pytorch tests, i thought it essentially disables [this](https://docs.wandb.ai/ref/python/run), but you are right - a this point `wandb` is mocked so there is nothing really to disable. Going to remove those.